wpcc.gemspec

```diff
@@ -20,7 +20,7 @@ Gem::Specification.new do |s|
 
   s.test_files = Dir['spec/**/*']
 
-  s.add_dependency 'dough-ruby'
+  s.add_dependency 'dough-ruby', '~> 5.25'
```

Comment:
@Tr4pSt3R I know that it is impratical, but that's the disavantage of using this way. We could move this dependency to the engine Gemfile and "leave the responsibility" to add a dough version to the main app (frontend repo). It is pratical but could leave with problems too, since you are just saying that the tool will work in *any* version of the tool.

By the way, searching on MAS, that's how is done in other tools: https://github.com/search?q=org%3Amoneyadviceservice+s.add_dependency+%22dough-ruby%22&type=Code